### ISSUE TITLE
[TODO] Installation LUCIDA sur VM Linux

### ISSUE THREAD

**Issue by Fernyassine** [Fernyassine/R-D-Lucida#1]
- Installation de Linux sur VM pour pouvoir tester LUCIDA
- Test de LUCIDA

**Comment by Fernyassine** [comment]
Quelques soucis lors de l'installation de LUCIDA sur Ubuntu 16.04. 

L'installation se fait simplement en lançant le "make" dans le dossier tools, mais des erreurs sont générées et le projet ne peut se lancer correctement ensuite.

De même pour Ubuntu 14.04. 

Nous allons tester avec d'autres OS.

**Comment by Fernyassine** [comment]
Downloading/unpacking distribute
  Cannot fetch index base URL https://pypi.python.org/simple/
  Could not find any downloads that satisfy the requirement distribute
Cleaning up...
No distributions at all found for distribute
Storing debug log for failure in /home/yassine/.pip/pip.log
Cannot fetch index base URL https://pypi.python.org/simple/
Could not find any downloads that satisfy the requirement pip in /usr/lib/python2.7/dist-packages
Downloading/unpacking pip
Cleaning up...
No distributions at all found for pip in /usr/lib/python2.7/dist-packages
Storing debug log for failure in /home/yassine/.pip/pip.log
Downloading/unpacking Flask==0.10.1 (from -r python_requirements.txt (line 1))
  Cannot fetch index base URL https://pypi.python.org/simple/
  Could not find any downloads that satisfy the requirement Flask==0.10.1 (from -r python_requirements.txt (line 1))
Cleaning up...
No distributions at all found for Flask==0.10.1 (from -r python_requirements.txt (line 1))
Storing debug log for failure in /home/yassine/.pip/pip.log
make[1]: *** [all] Error 1
make[1]: Leaving directory `/home/yassine/Desktop/lucida-master/tools'
make: *** [local] Error 2

**Comment by zahirBadr** [comment]
comme indiquer dans le dépot de lucida, il est conseillé d'utiliser le ubuntu 14.04 en 64bits, ton problème est du au proxy tu dois re-paramétrer ton réseau  (connecte toi sur un wifi sans proxy)

**Comment by Fernyassine** [comment]
Yep, je confirme ça passe mieux sur un réseau sans proxy ! Thanks 👍

**Comment by zahirBadr** [comment]
pensez à réserver une mémoire swap suffisante pour éviter toute problème de mémoire (output of memory).

**Comment by Fernyassine** [comment]
L'installation se passe mieux lorsqu'on installe les modules séparément. Mais il y a un soucis avec OpenCV qui fait planter la VM.
